config/400-webhook-service.yaml

```diff
@@ -21,8 +21,8 @@ metadata:
   name: webhook
   namespace: cloud-run-events
 spec:
+  selector:
+    role: cloud-run-events-webhook
```

Comment:
`webhook`

Reply:
400-controller.yaml uses the [control-plane: cloud-run-events-controller](https://github.com/GoogleCloudPlatform/cloud-run-events/blob/8da14eb2bc52d58c256a884e11205645dcc91e66/config/400-controller-service.yaml#L22), I would like standardize on one style. My preference is to prefix with `cloud-run-events-`, but could be convinced to omit it.

Although, now that I look again, I should change the name of the label to match the controller too.

Reply:
I want just `webhook`.